tutorials/containers/use-cuda-within-lxd/use-cuda-within-lxd.md

```diff
@@ -18,15 +18,22 @@ Duration: 3:00
 
 Hugely parallelised GPU data processing, using either [CUDA][nvidiacuda] or [OpenCL][opencl], is changing the shape of data science. It even has its own snappy acronym - GPGPU - General-purpose computing on graphics processing units.  It's no surprise, then, that flexible, scalable access to these GPU resources is becoming a key requirement in many cloud deployments (see the Canonical Distribution of [Kubernetes][kubernetes] for a good example). But it's not always easy, nor cheap, to get started. Unless you use [LXD][lxd].
 
-LXD's unrivalled density in real-world cloud deployments, and its ability to run locally, make it a game-game changing tool for experimenting with cloud-like GPU data processing. It enables you to create local scalable deployments using nothing more than a PC with a GPU or two. As we'll now demonstrate.
+![cuda-lxd-logos](https://assets.ubuntu.com/v1/8546712c-rect5436.png)
 
-## Requirements
-Duration: 2:00
+LXD's unrivalled density in real-world cloud deployments, and its ability to run locally, make it a game-changing tool for experimenting with cloud-like GPU data processing. It enables you to create local scalable deployments using nothing more than a PC with a GPU or two. As we'll now demonstrate.
+
+### What you'll learn
+
+- How to replace default Nvidia drivers with the latest ones
+- How to install the CUDA toolkit
+- How to configure LXD to use Nvidia GPUs and CUDA
+
+### What you'll need
 
 Our configuration is going to be based on the following:
 - 1 or more Nvidia GPUs
 - [Ubuntu 16.04 LTS][xenial] (Xenial)
-- LXD version 2.5 or higher (2.13 is the current Xenial version)
+- [LXD](https://tutorials.ubuntu.com/tutorial/tutorial-setting-up-lxd-1604) version 2.5 or higher (2.13 is the current Xenial version)
```

Comment:
Just a minor comment; the other links are in their own section at the end of the document.